app/web/template/choose_attorneys.gohtml

```diff
@@ -16,10 +16,22 @@
 
             {{ template "input" (input . "first-names" "firstNames" .Form.FirstNames "classes" "govuk-input--width-20") }}
             {{ template "input" (input . "last-name" "lastName" .Form.LastName "classes" "govuk-input--width-20") }}
-            {{ template "input" (input . "email" "email" .Form.Email "classes" "govuk-input--width-20" "type" "email") }}
+            {{ template "input" (input . "email" "email" .Form.Email "classes" "govuk-input--width-20" "type" "email" "spellcheck" "false" "autocomplete" "email") }}
```

Comment:
just following the advice on https://design-system.service.gov.uk/patterns/email-addresses/